generator/js_generator.cc

```diff
@@ -3648,13 +3677,22 @@ void Generator::GenerateFile(const GeneratorOptions& options,
           "    Function('return this')();\n\n");
     }
 
-    for (int i = 0; i < file->dependency_count(); i++) {
-      const std::string& name = file->dependency(i)->name();
-      printer->Print(
-          "var $alias$ = require('$file$');\n"
-          "goog.object.extend(proto, $alias$);\n",
-          "alias", ModuleAlias(name), "file",
-          GetRootPath(file->name(), name) + GetJSFilename(options, name));
+    if (options.import_style == GeneratorOptions::kImportEs6) {
+      for (int i = 0; i < file->dependency_count(); i++) {
+        const std::string& name = file->dependency(i)->name();
+        printer->Print("import * as $alias$ from '$file$';\n"
+                       "goog.object.extend(proto, $alias$);\n",
```

Comment:
`goog.object.extend` is deprecated, we should just use `Object.assign`.
Although I'm a little curios what's getting extended here. If this is building up a goog.module style namespace containing all protobufs, that's probably not something we want.